packages/react/src/analytics/integrations/Vitorino/providers/Forter/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const ForterTokenReadyEvent = 'ftr:tokenReady';
+export const ForterTokenTid = 1364;
+export const ForterTokenLoadedAnalyticsEvent = 'forterTokenLoaded';
+
+// site ID for forter sandbox - https://portal.forter.com/api/general/introduction
+export const ForterSandboxSiteId = 'b2f674e65110';
+// site ID for forter production - https://portal.forter.com/api/general/introduction
+export const ForterProductionSiteId = '0c0275c12bca';
```

Comment:
Same as with the Riskified comment, this should not be exposed in source code. Have the user pass it and document which values he should use.

Reply:
Update: It was removed the defaults, as we discuss offline.